stan/math/fwd/meta.hpp

```diff
@@ -1,11 +1,10 @@
 #ifndef STAN_MATH_FWD_META_HPP
 #define STAN_MATH_FWD_META_HPP
 
+#include <stan/math/fwd/scal/meta/ad_promotable.hpp>
```

Comment:
Yes, just like in the prim/meta.hpp, ad_promotable.hpp comes at the top. Good catch.

Reply:
Great, thanks!